examples/angular/src/app/lib/layout/row/row.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+import { ColumnComponent } from '../column/column.component';
+import { DotPageAssetLayoutRow } from '../../models';
+
+@Component({
+  selector: 'dotcms-row',
+  standalone: true,
+  imports: [ColumnComponent],
+  template: `@for(column of row.columns; track $index) {
+    <dotcms-column [column]="column" />
+  }`,
+  styleUrl: './row.component.css',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RowComponent {
+  @Input() row!: DotPageAssetLayoutRow;
```

Comment:
Why no signal? any reason?

Reply:
`Singal Inputs` are still in dev-preview, since this is an example, I thought we should use normal inputs (Until Signal Inputs are released). Thoughts @fmontes @oidacra?

- [Signal Inputs](https://angular.io/api/core/InputSignal)

Reply:
For me, can use a decorator input.  It is an example.